### DISCUSSION TITLE
Adding support to 6 channel 'alternative' hub from

### DISCUSSION THREAD

**Discussion by dwadwa3g** [pybricks/support#1478] [Ideas]
Do you think it is possible to add support for this type of hubs?
https://ae01.alicdn.com/kf/S087749bbbb4f49fda161831d08f50443Y/Techniczne-cz-ci-silnika-Multi-Power-Functions-8293-8883-narz-dzie-serwo-silnik-poci-gu-zestawy.jpg_640x640.jpg_.webp

https://pl.aliexpress.com/item/1005001920536750.html?spm=a2g0o.order_list.order_list_main.127.25a71c24F2gYjn&gatewayAdapt=glo2pol

**Comment by laurensvalk** [comment]
Supporting all official LEGO hardware is probably enough of a challenge as it is :)
